alpaca/rest.go

```diff
@@ -223,13 +223,13 @@ func (c *client) GetAccountActivities(activityType *string, opts *AccountActivit
 			q.Set("activity_types", strings.Join(*opts.ActivityTypes, ","))
 		}
 		if opts.Date != nil {
-			q.Set("date", opts.Date.String())
+			q.Set("date", opts.Date.UTC().Format(time.RFC3339))
```

Comment:
One more comment: this removes support from subsecond data parameters, as pointed out by your unit test :) Please use RFC3339Nano instead!

Reply:
Interesting, I didn't know you could send RFC3339Nano to Alpaca. Consider it done! :)